easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -553,21 +553,29 @@ def update(self, key, value, allow_duplicate=True):
         """
         Update a string configuration value with a value (i.e. append to it).
         """
+        if isinstance(value, string_type):
+            lval = [value]
+        elif isinstance(value, list):
+            lval = value
+        else:
+            msg = "Can't update configuration value for %s, because the "
+            msg += "attempted update value, '%s', is not a string or list."
+            raise EasyBuildError(msg, key, value)
+
         prev_value = self[key]
```

Comment:
`prev_value` should be renamed now that we're updating it in place?

Reply:
any suggestions? new_value?

Reply:
`param_value`?